lib/Items/ApplicationDockItem.vala

```diff
@@ -414,19 +418,45 @@ namespace Plank
 				item.activate.connect (() => pin_launcher ());
 				items.add (item);
 			}
-			
+
 			var event_time = Gtk.get_current_event_time ();
 			if (is_running () && window_count > 0) {
 				var item = create_menu_item ((window_count > 1 ? _("_Close All") : _("_Close")), "window-close-symbolic;;window-close");
 				item.activate.connect (() => WindowControl.close_all (App, event_time));
 				items.add (item);
 			}
-			
+
+			if (!is_running () && switcheroo_control != null && switcheroo_control.has_dual_gpu) {
+
+				print("launcher: %s\n", Prefs.Launcher);
```

Comment:
```suggestion
```

I would suggest removing this, or at least converting it to a `debug ()` with a more clear message of what it's actually debugging.